packages/react-reconciler/src/__tests__/ReactFlushSyncNoAggregateError-test.js

```diff
@@ -140,6 +142,6 @@ describe('ReactFlushSync (AggregateError not available)', () => {
     // AggregateError is not available, React throws the first error, then
     // throws the remaining errors in separate tasks.
     expect(error).toBe(aahh);
-    expect(flushFakeMicrotasks).toThrow(nooo);
+    await flushFakeMicrotasks();
```

Comment:
Should we assert somewhere that this was thrown to reportError or whatever?

Reply:
So it's a bit weird about what we should be testing here. Because this is to test that something works even if aggregated errors is not supported by the platform. However, we only do aggregation at all in `act()` now. So we're testing `act()` and if you use `act` in an environment that doesn't support aggregated errors, then this does get dropped. Not logged somewhere else.

Which I guess is mostly if you use act in a non-JSDOM environment.

Reply:
Makes sense, we should probably update the comment on 143 then